src/org/opendatakit/briefcase/ui/reused/source/RemoteServerDialog.java

```diff
@@ -60,6 +61,7 @@ private RemoteServerDialog(RemoteServerDialogForm form, RemoteServer.Test server
             }
           }
           form.unsetTestingConnection();
+          form.setCursor(Cursor.getPredefinedCursor(Cursor.DEFAULT_CURSOR));
```

Comment:
Maybe I'm a little bit confused, but is this really needed here?

I mean, you're already dealing with changing to the wait cursor and back to the default cursor in the `RemoteServerDialogForm.java`.

Reply:
Yes, but the changes in `RemoteServerDialogForm.java` only resolves the `BriefcaseException` and `MalformedURLException`, unexpected errors like `connection time out` cannot be covered. The change here is for those unexpected.

Reply:
OK, I understand now, although we shouldn't really interact directly with the UI (use Swing stuff) outside the `*Form` classes. 

Do you think we could hide resetting the cursor inside the `RemoteServerDialogForm.unsetTestingConnection()` method?

Maybe even setting it to waiting in the `RemoteServerDialogForm.setTestingConnection()` method.

Reply:
Haha, you're right, let me do the refactor and test now :)